package.json

```diff
@@ -19,10 +19,10 @@
   ],
   "dependencies": {
     "aes-js": "^3.1.2",
-    "bfx-api-node-core": "^1.5.5",
+    "bfx-api-node-core": "git+https://github.com/avsek477/bfx-api-node-core.git#orders-models-update",
```

Comment:
needs to be changed after this [PR](https://github.com/bitfinexcom/bfx-api-node-core/pull/29) is merged.

Reply:
```suggestion
    "bfx-api-node-core": "^1.5.6",
```